### ISSUE TITLE
ts-loader errors on build do not show filename

### ISSUE THREAD

**Issue by ervoiox** [wmonk/create-react-app-typescript#14]
On `npm build`, If you have a build error with Typescript, ts-loader does not show the file paths.

To demonstrate,

1. `npm uninstall @types/react --save`
2. `npm run build`

```
> react-scripts-ts build

Creating an optimized production build...
ts-loader: Using typescript@2.1.5 and /home/user/ts-demo/tsconfig.json
Failed to compile.

(11,43): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

(7,3): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

(9,7): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(9,7): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(10,9): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(10,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(11,11): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(11,11): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(12,11): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(12,11): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(12,31): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(13,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(14,9): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(14,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(15,32): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

(15,32): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(15,49): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(16,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(17,7): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

(7,19): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.
```

Where as, if you run `npm start`:

```
Failed to compile.

Error in /home/user/ts-demo/node_modules/@types/react-dom/index.d.ts
(11,43): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

Error in ./src/index.tsx
(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

Error in ./src/index.tsx
(7,3): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

Error in ./src/App.tsx
(9,7): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(9,7): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(10,9): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(10,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(11,11): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(11,11): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(12,11): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(12,11): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(12,31): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(13,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(14,9): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(14,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(15,32): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.

Error in ./src/App.tsx
(15,32): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(15,49): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(16,9): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in ./src/App.tsx
(17,7): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists

Error in /home/user/ts-demo/src/App.test.tsx
(1,24): error TS7016: Could not find a declaration file for module 'react'. '/home/user/ts-demo/node_modules/react/react.js' implicitly has an 'any' type.

Error in /home/user/ts-demo/src/App.test.tsx
(7,19): error TS2602: JSX element implicitly has type 'any' because the global type 'JSX.Element' does not exist.
```

You can see the file paths. Any ideas?

**Comment by wmonk** [comment]
Huh that's weird, might be one for the typescript-loader. Will have a look into it.

**Comment by wmonk** [comment]
Fixed in #15 and released 🎉
